resources/schemas/cds.xml

```diff
@@ -1412,4 +1412,84 @@
             <column columnName="container"/>
         </columns>
     </table>
+    <table tableName="import_virus_metadata_all" tableDbType="TABLE">
+        <columns>
+            <column columnName="cds_virus_id"/>
+            <column columnName="virus"/>
+            <column columnName="virus_full_name"/>
+            <column columnName="virus_backbone"/>
+            <column columnName="virus_host_cell"/>
+            <column columnName="virus_plot_label"/>
+            <column columnName="virus_type"/>
+            <column columnName="virus_species"/>
+            <column columnName="clade"/>
+            <column columnName="neutralization_tier"/>
+            <column columnName="container"/>
+        </columns>
+    </table>
+    <table tableName="virus_metadata_all" tableDbType="TABLE">
+        <columns>
+            <column columnName="cds_virus_id"/>
+            <column columnName="virus"/>
+            <column columnName="virus_full_name"/>
+            <column columnName="virus_backbone"/>
+            <column columnName="virus_host_cell"/>
+            <column columnName="virus_plot_label"/>
+            <column columnName="virus_type"/>
+            <column columnName="virus_species"/>
+            <column columnName="clade"/>
+            <column columnName="neutralization_tier"/>
+            <column columnName="container"/>
+        </columns>
+    </table>
+    <table tableName="import_virus_lab_id" tableDbType="TABLE">
+        <columns>
+            <column columnName="rowId"/>
```

Comment:
cds typically don't use rowId as far as I remember, is there a specific reason to use one here?

Reply:
that's true, I had added that before Jason gave me the final schema & FK's